tests/server/master_server/metadata_manager_unit_test.cc

```diff
@@ -8,32 +8,45 @@ using namespace gfs::server;
 
 class MetadataManagerUnitTest : public ::testing::Test {
  protected:
-  void SetUp() override { metadataManager_ = MetadataManager::GetInstance(); }
+  void SetUp() override { metadata_manager_ = MetadataManager::GetInstance(); }
 
-  MetadataManager* metadataManager_;
+  MetadataManager* metadata_manager_;
 };
 
 // Helper function to join a colletion of threads and cleanup the container for
 // these threads
-void joinAndClearThreads(std::vector<std::thread>& threads) {
+void JoinAndClearThreads(std::vector<std::thread>& threads) {
   for (auto& t : threads) {
     t.join();
   }
   threads.clear();
 }
 
+// Helper function to compute a filename given a base string and a nested 
+// level. The name scheme takes the following form:
+// "/{base}0/{base}1/..../{base}{nested_level-1}"
+std::string ComputeNestedFileName(const std::string& base, 
+                                  const int nested_level) {
+  std::string filename;
+  for (int i = 0; i < nested_level; i++) {
```

Comment:
FYI. This is not performance optimal since this will involve a lot of string concatenations. however, since this is a test, it's fine; but in server code, it's best to use a string builder. no changes necessary here

Reply:
Correct, this is essentially O(N^2) as we sweep over all nested level. But just for testing purposes.